config/services.php

```diff
@@ -16,12 +17,17 @@
 use Baldinof\RoadRunnerBundle\RoadRunnerBridge\HttpFoundationWorker;
 use Baldinof\RoadRunnerBundle\RoadRunnerBridge\HttpFoundationWorkerInterface;
 use Baldinof\RoadRunnerBundle\Worker\Dependencies;
+use Baldinof\RoadRunnerBundle\Worker\GrpcWorker;
+use Baldinof\RoadRunnerBundle\Worker\GrpcWorkerInterface;
 use Baldinof\RoadRunnerBundle\Worker\Worker;
 use Baldinof\RoadRunnerBundle\Worker\WorkerInterface;
 use Psr\Log\LoggerInterface;
 use Spiral\Goridge\RPC\RPCInterface;
 use Spiral\RoadRunner\Environment;
 use Spiral\RoadRunner\EnvironmentInterface;
+use Spiral\RoadRunner\GRPC\Invoker as GrpcInvoker;
+use Spiral\RoadRunner\GRPC\Server;
+use Spiral\RoadRunner\GRPC\ServiceInterface;
```

Comment:
personal preference: I would add aliases for Server and ServiceInterface: GrpcServer and GrpcServiceInterface to keep things consistent and avoid future collisions

Reply:
You are right! It's done.